extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -159,6 +176,19 @@ export class AmpSidebar extends AMP.BaseElement {
     }, true);
   }
 
+  /** @override */
+  onLayoutMeasure() {
+    // Check our toolbars for changes
+    this.toolbars_.forEach(toolbar => {
+      toolbar.checkToolbar(() => this.closeIfOpen_());
```

Comment:
This applies the toolbar dependent on the media query and viewport.

I was trying to think of a more verbose name than checkToolbar(), but had trouble coming up with one. I am open to some suggestions 😄